README.md

```diff
@@ -95,6 +95,31 @@ Each example performs the following steps:
 - Perform language-specific environment setup
 - Run the example
 
+### Exploring the Conjur API with Postman
+
+#### 1. Start a local Conjur server
+
+Use the provided script `bin/start_conjur`.
+
+#### 2. Setup Postman environment
+
+Use the provided script `postman/setup_postman`.
+
+This script generates a Postman Collection, as well as an 
+importable Postman Environment of important variables
+
+#### 3. Import the collection and environment into Postman
+
+These are saved under the filenames `postman/collection.json` and `postman/env.json`, respectively.
+
+Postman documentation for importing a collection is found
+[here](https://learning.postman.com/docs/getting-started/importing-and-exporting-data/#importing-postman-data).
+
+#### 4. Make a request
+
+Be sure to read each endpoint's accompanying documentation for required parameters and 
+authentication schemes.
+
```

Comment:
Can we add an example here? There was just recently some questions about this in cyberark/conjur-quickstart#40 - I think if we can add an example of some specific steps to take, it would be great.

If it feels like too much for the README (like it's going to get obnoxiously long with this example), then one technique you can use is to put the `Example` in a collapsible section. We use the technique in the [Secretless demo README](https://github.com/cyberark/secretless-broker/tree/master/demos/k8s-demo)

Reply:
thank you for adding more content! is there anything we can say here about how users should get a formatted token?

Reply:
The `{{formatted_token}}` collection variable is updated on every call by a pre-request script, so getting and formatting a token isn't on the user, which is described to them in [step 2](https://github.com/cyberark/conjur-openapi-spec/pull/147/files?file-filters%5B%5D=.md&file-filters%5B%5D=.py&file-filters%5B%5D=No+extension#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R113-R115).

Reply:
Also, the MD for the collapsible section is being really strange, and assumes the whole section should be a code block for some reason. Haven't found the cause, but figured it might be a good idea to move the example docs to `examples/postman/README.md` and link to it from `README.md` instead of the collapsible section.

Reply:
I think the text could be a little bit clearer about how the pre-request script is working - it says there is a utility script and then the examples have a bunch of variables wrapped in double curly braces `{{ var }}` - the connection isn't clear

Reply:
Added some more context - referencing variables with braces, which variables are included in setup, a link to Postman docs on pre-request scripts and a more specific description of the script included in setup ([link](https://github.com/cyberark/conjur-openapi-spec/pull/147/files#diff-2bf6b8eee614d23ccc929c3848ce42111e73de97ebacb56cd46105bd7588775dR11-R32))